tools/src/main/scala/com/nvidia/spark/rapids/tool/EventLogPathProcessor.scala

```diff
@@ -183,11 +183,16 @@ object EventLogPathProcessor extends Logging {
         Map.empty[EventLogInfo, Long]
       }
       matched.take(numberofEventLogs)
-    }.getOrElse(matchedLogs)
-
+    } else {
+      matchedLogs
+    }
     filteredLogs.keys.toSeq
   }
 
+  def filterByAppName(filterNLogs: Option[String]): Boolean = {
```

Comment:
the name here seems wrong since its either by app name or by app time, perhaps use something like filerByAppCriteria